extensions/lifecycle/objectProcessor/policy_transition.json

```diff
@@ -0,0 +1,15 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "LifecycleExpirationTransitionProcessor",
```

Comment:
The name seems off

Reply:
yes. fixed `LifecycleObjectTransitionProcessor`